index.html

```diff
@@ -0,0 +1,525 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" href="style.css">
+      <script src="https://kit.fontawesome.com/6e699a3632.js" crossorigin="anonymous"></script>
+      <title>Space Reveals Big Bang's Smoking gun</title>
+   </head>
+   <body>
+      <header>
+         <nav>
+            <ul class="nav_links">
+               <li><a href="#"><i class="fas fa-search"></i></a></li>
+               <li><a href="#"><i class="fas fa-bars"></i></a></li>
+               <li><a href ="#" class="Spacelink"><strong>SPACE & COSMOS</strong></a></li>
+            </ul>
+            
+         </nav>
+         <a class="logo" href="#"><img src="https://www.vectorlogo.zone/logos/nytimes/nytimes-wordmark.svg" alt="nytimeslogo"></a>  
+         <button class="btn" type="submit">SUBCRIBE NOW</button>
+         <button class= "btn" type="submit">LOG IN</button> 
+      </header>
```

Comment:
## Improve the styling on the header
- [ ] Kindly remove the spacing between the `subscribe` and the `login` buttons.  I would suggest you place them in the same div and only work on the margin styling inside this div. thereby, you will have them no far spaced out in the main navbar. 
- [ ] Do the same for the  `hamburger` and the `search` icon as well
- [ ] Ideally, the layout of the navbar should resemble the original navbar as contrasted below.

|[Source page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone page](https://raw.githack.com/JulianaOsemeke/NewYorkTimesArticle/Solo-Project/index.html)|
|:--:|:--:|
|![header](https://user-images.githubusercontent.com/22345961/108258998-ae08fd80-7171-11eb-8c4b-b0999c9053c9.png)|![header-clone](https://user-images.githubusercontent.com/22345961/108259023-b5300b80-7171-11eb-863d-10dea6cf5d09.png)|